lib/npack.js

```diff
@@ -209,11 +209,17 @@ exports.install = function(options, callback) {
 		function() {
 			logger.log('Sync temp package npm dependencies');
 
-			// sync node_modules
-			npmUtils.sync({
+			var syncOptions = {
```

Comment:
We should do something with "Copy current package node_modules to temp package" which is run before sync beacuse with npm ci it's not needed.